lib/embulk/input/marketo/lead.rb

```diff
@@ -42,35 +42,38 @@ def self.generate_columns(metadata)
         end
 
         def run
-          count = 0
           from_datetime = task[:from_datetime]
-          to_datetime = task[:to_datetime]
+          to_datetime = task[:to_datetime] || Time.now
+          ranges = task_target(from_datetime, to_datetime, task[:workers], index)
```

Comment:
Hmm...Why should transaction method store time ranges to task? time ranges are always same generated from `from_datetime` and `to_datetime`.